prov/efa/src/efa_domain.c

```diff
@@ -263,6 +266,7 @@ int efa_domain_open(struct fid_fabric *fabric_fid, struct fi_info *info,
 		goto err_free;
 	}
 
+	dlist_insert_tail(&efa_domain->list_entry, &g_efa_domain_list);
```

Comment:
there could be multiple efa domains ?

Reply:
I guess it is for the convenience of getting efa domain from `g_efa_domain_list` in the fork callback.

Reply:
Yes. there can be multiple efa domains. 

Application can create multiple domains. In fact, Open MPI create 2 domains: one for mtl/ofi, and other for btl/ofi.
Also aws-ofi-nccl create one domain for each NIC.

Reply:
I see, thanks!